src/views/Send/Send.vue

```diff
@@ -36,6 +36,7 @@
                 placeholder="Address"
                 autocomplete="off"
                 required
+                @input="getDomainAddress"
```

Comment:
Can you add `debounce` to this? It makes a network call and it would be slow an expensive to call it for every character the user edits in the field

Reply:
It won't call network on every character edit, it only call if the domain has a valid TLD

Reply:
Even for valid TLD it shouldn't be calling per change, debounce is a good practice for input calling some kind of async. In the future we cannot assume what this function will be doing under the hood. For example if implementation changes, or other name resolver.